.github/workflows/firestore.yml

```diff
@@ -369,7 +403,7 @@ jobs:
     runs-on: ubuntu-latest
     if: always()
     name: Check all required Firestore tests results
-    needs: [cmake, cmake-prod-db, xcodebuild, spm]
+    needs: [cmake, cmake-prod-db, xcodebuild, spm-source, spm-binary]
```

Comment:
Interesting– the `check-required-tests` was running indefinitely and was blocking merging. The issue was that it _needed_ the `spm` to finish but the `spm` job was renamed so it was waiting for a job that doesn't exist to finish. It makes sense, but is kind of sneaky.

cc: @paulb777 @wu-hui